### ISSUE TITLE
pysideuic workaround not working on OS Artix Linux

### ISSUE THREAD

**Issue by d1nuc0m** [boltsparts/BOLTSFC#35]
Hello,
I am trying to use BOLTS 0.4 (alias downloaded this repository) on FreeCAD 0.19 rev. 24291, OS Artix Linux Plasma-Runit.
I had practically the same problems of issue #28, so I tried to use the workaround. It didn't work:

```
Traceback (most recent call last):
  File "/home/d1nuc0m/.FreeCAD/Macro/start_bolts.FCMacro", line 1, in <module>
    import BOLTS
  File "/usr/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "/home/d1nuc0m/.FreeCAD/Macro/BOLTS/__init__.py", line 44, in <module>
    from .gui import freecad_bolts as boltsgui
  File "/usr/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "/home/d1nuc0m/.FreeCAD/Macro/BOLTS/gui/freecad_bolts.py", line 34, in <module>
    from . import uic
  File "/usr/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "/home/d1nuc0m/.FreeCAD/Macro/BOLTS/gui/uic.py", line 2, in <module>
    from pyside2uic import compileUi
  File "/usr/lib/python3.9/site-packages/shiboken2/files.dir/shibokensupport/__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
<class 'ModuleNotFoundError'>: No module named 'pyside2uic'
```
Here are the two files modified according to the workaround: [Modified Files.zip](https://github.com/boltsparts/BOLTSFC/files/6758771/Modified.Files.zip)

**Comment by berndhahnebach** [comment]
https://github.com/boltsparts/BOLTSFC/issues/40

**Comment by d1nuc0m** [comment]
> have you seen ... [#27 (comment)](https://github.com/boltsparts/BOLTSFC/issues/27#issuecomment-712316909)

Thank you, yes, I had seen it, it links to #28 
It looks like the problem has been solved by #40 , I'll let you know if there will be problems on a "stock" install of OS and FreeCAD
